### PR TITLE
New Port Lisa The Bashful

### PR DESCRIPTION
**Pull request by Tlaggypyro**
# New Port for { Lisa the Bashful }

## Game Information
- **Title**: { Lisa The Bashful }
- **URL**: {https://gamejolt.com/games/Lisathebashful/430290 }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/lisathebashful/port.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "aarch64"
+        ],
+        "desc_md": null,
+        "inst_md": null
+    },
+    "status": {
```
Comment (kloptops) [inline ports/lisathebashful/port.json:30]: The "status" and "files" tags shouldn't be included.

**Review by kloptops — CHANGES_REQUESTED** [review]
Just need the port.json changed.

File: ports/lisathebashful/LISA The Bashful.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+get_controls
+
+GAMEDIR=/$directory/ports/lisathebashful
+CONFDIR="$GAMEDIR/conf/"
+
+CUR_TTY=/dev/tty0
+$ESUDO chmod 666 $CUR_TTY
+
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```
Comment (kloptops) [inline ports/lisathebashful/LISA The Bashful.sh:27]: Outdated logging directive, use `> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1`

**Review by kloptops — CHANGES_REQUESTED** [review]
Just need to change the logging line in the bash script.

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]

**Pull request merged**